c10/core/Scalar.h

```diff
@@ -31,7 +31,7 @@ class C10_API Scalar {
   // We can't set v in the initializer list using the
   // syntax v{ .member = ... } because it doesn't work on MSVC
 
-  AT_FORALL_SCALAR_TYPES(DEFINE_IMPLICIT_CTOR)
+  AT_FORALL_SCALAR_TYPES_WITHOUT_BOOL(DEFINE_IMPLICIT_CTOR)
```

Comment:
If we have Bool tensors it seems to make sense that we have a constructor here that takes a bool.  Does adding one cause problems?

Reply:
if i add bool here, im getting this kind of errors: 
jit/symbolic_variable.h:79:20: error: ambiguous overload for ‘operator*’ (operand types are ‘const torch::jit::SymbolicVariable’ and ‘torch::jit::Value*’)

_jit/symbolic_variable.h:126:20: error: ambiguous overload for ‘operator+’ (operand types are ‘const torch::jit::SymbolicVariable’ and ‘torch::jit::Value*’)

Im not sure if im supposed to be touching this. should i?